### ISSUE TITLE
Input Shape of the Pose Model

### ISSUE THREAD

**Issue by heigex** [googlecreativelab/teachable-machine#51]
I trained a Pose Model for two classes. I trained it for experimenting on some random labels. I want to run inference on this `tfjs` model but the model accepts `float32[?,14739]` as input. Usually, all the image models accept input in these formats:

1. Tensorflow format: `[batch_size, height, width, channels]`
2. PyTorch format: `[batch_size, channels, height, width]`

The model doesn't accept input in any of the above format. Am i missing something here?

**How did I check the input shape?**
1. Download the `*.zip` file of the model
2. Unzip the file.
3. Open [netron.app](netron.app)
4. Upload the `model.json` file on the [netron.app](netron.app)

Link to Pose Model: https://teachablemachine.withgoogle.com/models/irJWnCc6c/
